main/arbitration_identifiers.h

```diff
@@ -0,0 +1,8 @@
+#include <stdint.h>
+
+static uint32_t send_identifier = 0x7E0; // ECU
+static uint32_t receive_identifier = 0x7E8; // ECU
```

Comment:
is it right for this to be static or is there no need?

Reply:
same deal with the static keyword being mostly used as a scope modifier in C - in a variable that already exists at the top level of the scope, it distinguishes between a "compilation unit" local variable and a truly global variable.

Reply:
it's OK and probably more correct - for most practical purposes `static` is just `private` to an execution unit when it's used at the top level in C (technically it moves a variable into the "local" link unit)